Move config logic into new workspace (#1042)
vltpkg/vltpkg: src/config/README.md

# @vltpkg/config

> Project config logic for `vlt`.

Reviewer: ```suggestion
Config utilities for the vlt client.
```

**[Usage](#usage)** · **[Functions](#functions)**

## Overview

> This is a tool that provides the project config logic used by the vlt
> cli.

Reviewer: ```suggestion
This are config utilities used by the vlt cli.
```
